docker/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
```

Comment:
@jmbabazi - one thing I noticed is that docker-compose version 3.0+ is out, and we might want to adopt that if we are establishing this new standard.  I also notice some things in other compose files I've seen, like "healthcheck" that seems interesting and like something we'd want to add.  You can see an example of that here:  https://github.com/openmrs-indianaems/openmrs-docker-indianaems/blob/master/app/docker-compose.override.yml